models/migrations/v145.go

```diff
@@ -47,9 +47,29 @@ func increaseLanguageField(x *xorm.Engine) error {
 			return err
 		}
 	case setting.Database.UseMSSQL:
+		// Yet again MSSQL just has to be awkward.
+		// Here we have to drop the constraints first and then rebuild them
```

Comment:
I think this is more a thing vor xorm?

Reply:
But Xorm doesn't handle it. Therefore we have to. 

Currently people on certain earlier versions of mssql cannot run Gitea doctor. We need to fix this.